### ISSUE TITLE
Modal dialog body getting focus when clicking radio button

### ISSUE THREAD

**Issue by davidrourke** [vmware/clarity#3642]
## Describe the bug

Clicking radio button within modal dialog is causing the body to get focus temporarily.  This does not happen with other controls.  

## How to reproduce

https://angular-w3svdn.stackblitz.io/

1.  Click on the Show Modal button.
2. Click between any of the radio button.

## Expected behavior

The body of the modal dialog show not get focus when clicking on a radio button.

## Versions

**App**

* Angular: 7.2.9
* Node: 10.14.2
* Clarity: 1.1.4

**Device:**

* Type: Macbook Pro
* OS: macOS Majave
* Browser Chrome
* Version 75.0.3770.142

## Additional notes

The problem does not exists in Clarity version 1.1.3

**Comment by davidrourke** [comment]
I see that this has been fixed in v2.2.1.  Do you plan on patching in v1.2.x?

Regards,
David

**Comment by coryrylan** [comment]
Yes this should be backported to 1.x. It missed the last release cycle so it should be backported soon.
